docs/UserDefinedLanguageServer.md

```diff
@@ -88,7 +88,13 @@ level Trace used in LSP console.
 
 ### Using template
 
-The `Template combo-box` provides some `language servers templates`, pre-filled with server name, command, mappings and potential configuration. 

```

Comment:
Why have you deleted this sentence?

Reply:
I will update it with 

The `Template combo-box` provides some default `language servers templates` located in [templates](https://github.com/redhat-developer/lsp4ij/tree/main/src/main/resources/templates) directory classpath, pre-filled with server name, command, mappings and potential configuration.

Reply:
I only modified it as new selector is not a combo-box, but a file selector. Diff seems to delete this and add my modified line instead of showing the difference on this line.